### ISSUE TITLE
[Idea] Tanker speed

### ISSUE THREAD

**Issue by HighwaymanEd** [dcs-liberation/dcs_liberation#468]
Hey guys, not sure if you've already solved this or not, but I had an idea about how to set the Tankers up. For argument's sake, let's assume that the tanker in question is a standard KC-135.

1. Set the tankers waypoints 1 & 2 and a racetrack orbit for point 1.
2. Set the speed of the tanker's orbit speed to the standard max endurance speed of a KC-135.
3. Set the tankers waypoints 3 & 4 to the same positions as 1 & 2 with a racetrack orbit for point 3.
4. Set the speed of the tanker's orbit speed to that required for an A-10.
5. Set the tankers waypoints 5 & 6 to the same positions as 1 & 2 with a racetrack orbit for point 5.
6. Set the speed of the tanker's orbit speed to that required for an F-16.
7. Once all airframes are accounted for, attach a trigger to switch the tanker's waypoint to waypoint 3 if an A-10 is in a 10-mile moving zone attached to the tanker, if it's an F-16, switch to waypoint 5, if all of the units are outside of the zone, switch back to waypoint 1

Finally, via a script, you can detect how much fuel a unit has, let me know if you need it. Using the script, when the Tankers fuel falls below 30% it triggers the takeoff of a 2nd tanker from its home airfield, then when that tanker is within the refueling area, trigger a flag that is used as an end condition for each of the orbits and have the first tanker RTB on its final waypoint.

Just an idea like I said, you've probably already figured all of this out, and I may have overlooked something that would cause issues, but I thought I'd send it anyway!

Finally, I've attached a NATO Air To Air Refuelling publication that may come in useful. (The interesting stuff starts around page 51 ;) )
[US_SRDATP 3.3.4.2(c).pdf](https://github.com/Khopa/dcs_liberation/files/5605800/US_SRDATP.3.3.4.2.c.pdf)

**Comment by dimkzr** [comment]
@DanAlbert while the idea described above sounds nice, can we get a simpler workaround in the meanwhile? Because with the KC-135 flying at 330kts, refueling the A-10C is impossible. We can either slow down all tankers to ~230kts, or add an additional one flying at that speed.

**Comment by cooljacob204** [comment]
Tankers are going way too fast in general to be useful at the moment. 

375kts at 20k feet.

I can't really use them in my f18 without using up all my fuel just chasing them down and it's impossible to use in the A10.

**Comment by KitchenDuty** [comment]
Yes i agree, useless in the a-10 for sure but even so with the 18. Mabye 16/14 can tank at these speeds.
